src/cljs/rems/catalogue.cljs

```diff
@@ -49,7 +49,16 @@
        (assoc ::catalogue catalogue)
        (dissoc ::loading-catalogue?))))
 
-(rf/reg-sub ::catalogue (fn [db _] (::catalogue db)))
+(rf/reg-sub ::full-catalogue (fn [db _] (::catalogue db)))
```

Comment:
Is this used anywhere? The ::catalogue sub could depend on this.

Reply:
This is used by the `change_resources.cljs` and since this is the main file where catalogue is used I thought it's better to have the subscription be here.

It's true that the subs could be chained but I think the case here is so trivial that maybe it's unnecessary.

Reply:
Now with chaining!